### ISSUE TITLE
Split pane from command prompt yields inconsistent results

### ISSUE THREAD

**Issue by tedhudek** [microsoft/terminal#4430]
# Environment

```none
Windows build number: 10.0.18363.0
Windows Terminal version: 0.9.271.0
```

# Steps to reproduce

1. Create desktop shortcut with `wt.exe ; split-pane ; split-pane`.
2. Run shortcut, observe Terminal's pane layout, close Terminal.  Repeat several times.
3. For me this has resulted in all of the following at different times
    * 3 panes with half vertical left and two quarters on the right
    * 3 panes with two quarters on left, half vertical on right
    * 2 vertical panes
    * once I even got a single pane

# Expected behavior

Expecting first bullet under step 3 above. Absent changes in screen dimension, command line invocation should produce same results each time.

# Actual behavior

Getting variable results rather than consistent results.  I do seem to get the expected one most often, but not 100% of the time.

**Comment by DHowett-MSFT** [comment]
This one fortunately has a quick workaround: specify `-V` or `-H` explicitly. :smile: Thanks for the report.

**Comment by FRANK3NST31N** [comment]
I have observed inconsistent results as well while specifying the -V or -H 

OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.18363 N/A Build 18363
wt Version:                 0.9.433.0

I followed the same procedure but used a script.

I created : test-terminal-split.CMD

File : test-terminal-split.CMD
Contents :

wt  ; split-pane -V  ; split-pane -H

then i execute this script several times.  

run 1 : two panes
run 2 : three panes OK
run 3 : two panes
run 4 : two panes
run 5 : three panes OK

**Comment by DHowett-MSFT** [comment]
I think this was fixed with #5090, for 0.11 (and soon, internal selfhost)

**Comment by DHowett-MSFT** [comment]
/dup #4618 (sorry to dupe you to a higher number, but that's the one closed by the PR :smile:)
